aptos-move/framework/aptos-framework/sources/randomness.move

```diff
@@ -81,7 +81,7 @@ module aptos_framework::randomness {
         let seed = *option::borrow(&randomness.seed);
 
         vector::append(&mut input, seed);
-        vector::append(&mut input, transaction_context::get_transaction_hash());
+        vector::append(&mut input, transaction_context::unique_session_hash());
```

Comment:
@igor-aptos , since `get_transaction_hash` is still available, should `get_transaction_hash` still be used in this release instead of `unique_session_hash`?

Reply:
Done updating `unique_session_hash` to just use `get_txn_hash`, and removing `unique_session_hash_internal`.

@igor-aptos , is it correct for the `randomness` module to use `unique_session_hash` instead of `get_transaction_hash` in this release?